content/blog/2018/07/2018-07-25-contributor-summit.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Join us at the Jenkins Contributor Summit San Francisco, Monday 17 September 2018"
+tags:
+- community
+- events
+- jenkins-world
+author: tracymiranda
+---
+
+The Jenkins Contributor summit is where the current and future contributors of the Jenkins project get together.
+This summit will be on Monday, September 17th 2018 in San Francisco, just before link:https://www.cloudbees.com/devops-world/san-francisco[DevOps World | Jenkins World].
+The summit brings together community members to learn, meet and help shape the future of Jenkins. 
+In the Jenkins commmunity we value all types and sizes of contributions and love to welcome new participants.
+Register link:https://www.eventbrite.com/e/jenkins-contributor-summit-san-francisco-tickets-48285340754[here].
+
+== Topics
+There are plenty of exciting developments happening in the Jenkins community. 
+The summit will feature a 'State of the Project' update including updates from the Jenkins officers. 
+We will also have updates on the 'Big 5' projects in active development:
+* link:https://jenkins.io/blog/2018/04/06/jenkins-essentials[Jenkins Essentials]
+* link:https://jenkins-x.io[Jenkins X]
+* link:https://www.praqma.com/stories/jenkins-configuration-as-code[Configuration as Code]
+* Jenkins Pipeline
+* Cloud Native Jenkins Architecture

```

Comment:
https://jenkins.io/sigs/cloud-native/ would be a good link

Reply:
good idea, done